### PR TITLE
Glove mode

### PR DESCRIPTION
**Pull request by leskal**
please test it.....

### CODE REVIEW

File: rootdir/init.qcom.rc [hunk 1]
```diff
@@ -98,6 +98,10 @@ on early-boot
     write /sys/module/pm8921_charger/parameters/enable_lowbatt_detect 1
 
 on boot
+    
+    # load custom kernel modules
+    insmod /system/lib/modules/cpufreq_wheatley.ko
```
Comment (Mrcl1450) [inline rootdir/init.qcom.rc]: Why this?

Comment (leskal) [inline rootdir/init.qcom.rc]: ooops sorry, remove it

Comment (leskal) [inline rootdir/init.qcom.rc]: move this: rootdir/system/etc/init.d/glove_mode_set   to  rootdir/system/bin/glove_mode_set.sh   i made a second pull request for moving it

File: cmhw/org/cyanogenmod/hardware/HighTouchSensitivity.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    /**
+     * This method allows to setup high touch sensitivity status.
+     *
+     * @param status The new high touch sensitivity status
+     * @return boolean Must be false if high touch sensitivity is not supported or the operation
+     * failed; true in any other case.
+     */
+    public static boolean setEnabled(boolean status) {
+        SystemProperties.set("hw.glove.mod.perm.w.on","0");
```
Comment (Dark-Passenger) [inline cmhw/org/cyanogenmod/hardware/HighTouchSensitivity.java]: Hi shouldn't you be getting rather than setting sys property also please change to bool

**Comment by ghost** [comment]
Hi cm supports this in 12.1 this looks like a very hacky way of doing it(sorry!) Maybe check out how its done for others ...

**Comment by leskal** [comment]
Bagyusz in cm12 rom for huashan use this method, I can't find another.... And for the change you mean to change boolean to bool?

**Comment by ghost** [comment]
Hi sorry about the bool value didn't read it thoroughly. My only problem is the use of scripts (very hacky) we have the kernel at our disposal let's use it. I think I saw Samsung jf implementing it officially.

**Comment by leskal** [comment]
because i'm newbie could tell me what to do?

**Comment by Mrcl1450** [comment]
See here http://review.cyanogenmod.org/#/c/73382/

**Comment by ghost** [comment]
to enable set value to 0 and to disable set to 1

/sys/devices/i2c-3/3-0024/main_ttsp_core.cyttsp4_i2c_adapter/signal_disparity

**Pull request closed without merging**